browser/src/map/Clipboard.js

```diff
@@ -501,12 +501,6 @@ L.Clipboard = L.Class.extend({
 	_getHtmlForClipboard: function() {
 		var text;
 
-		if ($('.ui-edit').is(':focus'))
-			return $('.ui-edit').value();
-
-		if ($('.w2ui-input').is(':focus'))
-			return $('.w2ui-input').value();
-
```

Comment:
Did you test if copying from comments still work? That kind of info would go to the commit message ideally. :-)

Reply:
Yep, tested just before sending the changes.